### PR TITLE
OV SIW setup labels (skip auth flow). Resolves OKTA-631851

### PR DESCRIPTION
**Pull request by vakaevich-okta**
## Description:

OV SIW setup labels (skip auth flow)

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-631851](https://oktainc.atlassian.net/browse/OKTA-631851)

### Reviewers:

@shuowu-okta , @yannongli-okta , @helenchen-okta , @nikhilvenkatraman-okta

### CODE REVIEW

**Review by helenchen-okta — APPROVED** [review]

File: packages/@okta/i18n/src/properties/login.properties [hunk 1]
```diff
@@ -1030,6 +1030,12 @@ oie.okta_verify.authenticator.description = Okta Verify is an authenticator app,
 oie.enroll.okta_verify.setup.title = Set up Okta Verify
 oie.enroll.okta_verify.setup.email.title = Check your email
 oie.enroll.okta_verify.setup.sms.title = Check your text messages
+oie.enroll.okta_verify.setup.skipAuth.subtitle = To set up Okta Verify on additional devices, you can copy an existing Okta Verify account onto a new device.
+oie.enroll.okta_verify.setup.skipAuth.openOv = Open Okta Verify on any of your other Okta Verify devices (<b>{0}</b>).
```
Comment (glenfannin-okta) [inline packages/@okta/i18n/src/properties/login.properties:1034]:
```suggestion
oie.enroll.okta_verify.setup.skipAuth.openOv = Open Okta Verify on any of your other Okta Verify devices (<$1>{0}</$1>).
```

File: packages/@okta/i18n/src/properties/login.properties [hunk 2]
```diff
@@ -1030,6 +1030,12 @@ oie.okta_verify.authenticator.description = Okta Verify is an authenticator app,
...
+oie.enroll.okta_verify.setup.skipAuth.selectAccount = In the app, select your account.
+oie.enroll.okta_verify.setup.skipAuth.addAccount = Choose <b>Add Account to Another Device.</b>
```
Comment (glenfannin-okta) [inline packages/@okta/i18n/src/properties/login.properties:1036]:
```suggestion
oie.enroll.okta_verify.setup.skipAuth.addAccount = Choose <$1>Add Account to Another Device.</$1>
```

**Comment by vakaevich-okta** [comment]
@glenfannin-okta the comments addressed, please review: https://github.com/okta/okta-signin-widget/pull/3317/commits/559a807ed26c9dcf7f1172d2b7a0cc0b37894455

**Review by glenfannin-okta — APPROVED** [review]

File: packages/@okta/i18n/src/properties/login.properties [hunk 3]
```diff
@@ -1030,6 +1030,12 @@ oie.okta_verify.authenticator.description = Okta Verify is an authenticator app,
 oie.enroll.okta_verify.setup.title = Set up Okta Verify
 oie.enroll.okta_verify.setup.email.title = Check your email
 oie.enroll.okta_verify.setup.sms.title = Check your text messages
+oie.enroll.okta_verify.setup.skipAuth.subtitle = To set up Okta Verify on additional devices, you can copy an existing Okta Verify account onto a new device.
+oie.enroll.okta_verify.setup.skipAuth.openOv = Open Okta Verify on any of your other Okta Verify devices (<$1>{0}</$1>).
+oie.enroll.okta_verify.setup.skipAuth.selectAccount = In the app, select your account.
+oie.enroll.okta_verify.setup.skipAuth.addAccount = Choose <$1>Add Account to Another Device.</$1>
```
Comment (yannongli-okta) [inline packages/@okta/i18n/src/properties/login.properties:1036]: nit: should we keep the period outside of the tag?

**Pull request merged**